src/OpenSSL/__main__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function
+
+import sys
+
+import OpenSSL.SSL
+import cryptography
+
+from . import version
+
+
+if __name__ != "__main__":
+    raise ImportError("This module can't be imported.")
+
+wot = """\
+pyOpenSSL: {pyopenssl}
+cryptography: {cryptography}
+OpenSSL: {openssl}
```

Comment:
You may want to distinguish between the linked version of OpenSSL and the compiled-against version.